flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/join/LookupJoinITCase.scala

```diff
@@ -208,15 +247,19 @@ class LookupJoinITCase(legacyTableSource: Boolean, isAsyncMode: Boolean) extends
 
   @Test
   def testLeftJoinTemporalTable(): Unit = {
-    val sql = s"SELECT T.id, T.len, D.name, D.age FROM T LEFT JOIN userTable " +
+    if (legacyTableSource) {
+      //Computed column do not support in legacyTableSource.
```

Comment:
You can use junit `Assume.assumeTrue()` instead.

PS, can we just create another table with computed column there and add test cases for this patch ? It is not necessary to add another test param which makes the test looks verbose and complex.